airflow/dag_processing/manager.py

```diff
@@ -485,51 +486,69 @@ def start(self):
         return self._run_parsing_loop()
 
     @provide_session
-    def _deactivate_stale_dags(self, session=None):
-        """
-        Detects DAGs which are no longer present in files.
-
-        Deactivate them and remove them in the serialized_dag table
-        """
+    def _scan_stale_dags(self, session=None):
```

Comment:
I think we can remove the session param (and @provide_session) from this method.

Reply:
Good point!